scenarios/tracking/02_evaluating_mot_challenge.ipynb

```diff
@@ -0,0 +1,463 @@
+{
+ "cells": [
```

Comment:
[minor] Consider rename the notebook to the shorter version "02_mot_challenge.ipynb" (totally fine that it does not train, but only evaluate)